src/main/resources/swagger.api/patron_permission.yaml

```diff
@@ -7,6 +7,7 @@ paths:
   /reading-room-patron-permission/{patronId}:
     parameters:
       - $ref: '#/components/parameters/userId'
+      - $ref: '#/components/parameters/servicePointId'
```

Comment:
servicePointId is not common to both get and put api

Reply:
fixed